charts/internal/calico/templates/typha-cpha/configmap-typha-horizontal-autoscaling.yaml

```diff
@@ -12,7 +12,7 @@ data:
       "nodesToReplicas":
       [
         [1, 1],
-        [10, 2],
+        [2, 2],
```

Comment:
Did you test this on a two node cluster? I hope that `maxUnavailable = 1` should allow a rolling update, but this requires on a two node cluster that an existing typha instance is shut down before a new one is created. Otherwise, the new one will stay in pending forever.

Reply:
Hm, isn't the `calico-typha-horizontal-autoscaler` Deployment rolled because the ConfigMap has changed? Because this deployment doesn't have a PDB.
Nevertheless, I tried out changing the config as shown above and a second `calico-typha-deploy` Pod was created, the first one kept running.

Reply:
My question was targeting the `calico-typha` deployment. The question was rather did the rollout of an update works on a two node cluster.